src/Operator/Constructors.jl

```diff
@@ -179,7 +280,7 @@ function GalleryOperator(
             collocatedquadrature = collocatedquadrature,
             mapping = mapping,
         )
-        return operatorgallery[name](basis, mesh)
+        return operatorgallery[name](basis, mesh, parameters = (wind = [1, 1]))
```

Comment:
```suggestion
        return operatorgallery[name](basis, mesh; parameters=parameters)
```

Reply:
I think we will likely need to change the other constructors, like mass and Poisson, to have `(...; parameters=nothing)`.